eslint.config.mjs

```diff
@@ -0,0 +1,131 @@
+import base, { createConfig } from '@metamask/eslint-config';
+import jest from '@metamask/eslint-config-jest';
+import nodejs from '@metamask/eslint-config-nodejs';
+import typescript from '@metamask/eslint-config-typescript';
+
+const config = createConfig([
+  {
+    ignores: ['dist/', 'docs/', '.yarn/'],
+  },
+
+  {
+    extends: base,
+
+    languageOptions: {
+      sourceType: 'module',
+      parserOptions: {
+        tsconfigRootDir: import.meta.dirname,
+      },
+    },
+
+    rules: {
+      // Consider copying this to @metamask/eslint-config
+      'jsdoc/require-jsdoc': [
+        'error',
+        {
+          require: {
+            // Classes
+            ClassDeclaration: true,
+            // Function declarations
+            FunctionDeclaration: true,
+            // Methods
+            MethodDefinition: true,
+          },
+          contexts: [
+            // Type interfaces that are not defined within `declare` blocks
+            ':not(TSModuleBlock) > TSInterfaceDeclaration',
+            // Type aliases
+            'TSTypeAliasDeclaration',
+            // Enums
+            'TSEnumDeclaration',
+            // Arrow functions that are not contained within plain objects or
+            // are not arguments to functions or methods
+            ':not(Property, NewExpression, CallExpression) > ArrowFunctionExpression',
+            // Function expressions that are not contained within plain objects
+            // or are not arguments to functions or methods
+            ':not(Property, NewExpression, CallExpression) > FunctionExpression',
+            // Exported variables at the root
+            'ExportNamedDeclaration:has(> VariableDeclaration)',
+          ],
+        },
+      ],
+      // Consider copying this to @metamask/eslint-config
+      'jsdoc/no-blank-blocks': 'error',
```

Comment:
I realized that this existed and I found this to be a helpful rule. It seemed that with the new JSDoc rules, running `yarn eslint --fix` added a bunch of empty JSDoc blocks to various symbols. With this enabled, I was able to quickly find those and fill them in.

Reply:
Cool, definitely in favour of moving this to `eslint-config`, but maybe release a fix with non-breaking changes first.

Reply:
Yeah, good point.